client/src/app/shared/shared-video-playlist/video-playlist.service.ts

```diff
@@ -188,14 +192,6 @@ export class VideoPlaylistService {
     return this.authHttp.post<{ videoPlaylistElement: { id: number } }>(url, body)
                .pipe(
                  tap(res => {
-                   const existsResult = this.videoExistsCache[body.videoId]
```

Comment:
Why did you remove this code?

Reply:
This breaks the "add to playlist" dropdown (video miniature menu for example)

Reply:
> I remember the caching makes the results in the "Add to playlist" dropdown inconsistent, depending on which playlists are loaded in the cache

Don't hesitate to create an issue with the reproducing steps so I can fix this behaviour.

>  By the way I just found out that the cache isn't cleared upon logout.

Thanks, I'll fix this!

> In which way does it break? When I test it seems to work.

https://user-images.githubusercontent.com/5180488/192965688-e4479f5c-77c3-4ca5-ba5a-219e7bec0541.mp4

> If we want to cache the result, isn't it better/easier to just cache the API result

The cache is more like a store, so we don't have to do API calls every time we add/remove a video from a playlist.